src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -1793,8 +1793,8 @@ internal List<CommandMetadata> GetRemoteCommandMetadata(out Dictionary<string, s
         private void WriteProgress(string statusDescription, int? percentComplete, int? secondsRemaining)
         {
             ProgressRecordType recordType;
-            if (secondsRemaining.HasValue && secondsRemaining.GetValueOrDefault() == 0 &&
-                percentComplete.HasValue && percentComplete.GetValueOrDefault() == 100)
+            if (secondsRemaining.GetValueOrDefault() == 0 &&
```

Comment:
The change alters the condition to evaluate to `true` when `secondsRemaining` is `null`.